src/metpy/calc/indices.py

```diff
@@ -104,10 +104,10 @@ def mean_pressure_weighted(pressure, *args, height=None, bottom=None, depth=None
     Parameters
     ----------
     pressure : `pint.Quantity`
-        Atmospheric pressure profile
+        Atmospheric pressure profile.
 
     args : `pint.Quantity`
-        Parameters for which the pressure-weighted mean is to be calculated
+        Parameters for which the weighted-continuous mean is to be calculated.
```

Comment:
Maybe instead "pressure-weighted continuous mean"?

Reply:
That makes sense to me, I've redone my local branch and made those changes to make it clearer, since we are still using pressure-weighting in some sense.